### PR TITLE
changelog 0.5.3

### PR DESCRIPTION
**Pull request by will-moore**
@francesw @jburel I'll get this released and bumped in IDR along with gallery if no objections?

### CODE REVIEW

**Comment by will-moore** [comment]
Oh dear - build failed, not due to this PR I think

```
#12 21.81  From       : /etc/pki/rpm-gpg/PGDG-RPM-GPG-KEY-RHEL
#12 22.40 Error: Failed to download metadata for repo 'pgdg12': repomd.xml GPG signature verification error: Bad GPG signature
#12 ERROR: process "/bin/sh -c dnf install -y -q     git &&     dnf clean all" did not complete successfully: exit code: 1

#11 [web 2/2] RUN dnf install -y -q     java-11-openjdk     git &&     dnf clean all
failed to solve: process "/bin/sh -c dnf install -y -q     git &&     dnf clean all" did not complete successfully: exit code: 1
#11 CANCELED
------
 > [omero 2/2] RUN dnf install -y -q     git &&     dnf clean all:
16.81  From       : /etc/pki/rpm-gpg/PGDG-RPM-GPG-KEY-RHEL
19.45 Importing GPG key 0x08B40D20:
19.45  Userid     : "PostgreSQL RPM Repository <pgsql-pkg-yum@lists.postgresql.org>"
19.45  Fingerprint: D4BF 08AE 67A0 B4C7 A1DB CCD2 40BC A2B4 08B4 0D20
19.45  From       : /etc/pki/rpm-gpg/PGDG-RPM-GPG-KEY-RHEL
21.81 Importing GPG key 0x08B40D20:
21.81  Userid     : "PostgreSQL RPM Repository <pgsql-pkg-yum@lists.postgresql.org>"
21.81  Fingerprint: D4BF 08AE 67A0 B4C7 A1DB CCD2 40BC A2B4 08B4 0D20
21.81  From       : /etc/pki/rpm-gpg/PGDG-RPM-GPG-KEY-RHEL
22.40 Error: Failed to download metadata for repo 'pgdg12': repomd.xml GPG signature verification error: Bad GPG signature
------
Warning: No resource found to remove for project "omeromapr".
```

**Comment by jburel** [comment]
Before doing a release, #92 should be considered since we now have email notification when pushing to [Pypi](https://pypi.org/)

**Comment by jburel** [comment]
This PR should also be considered https://github.com/ome/omero-mapr/pull/91

**Comment by will-moore** [comment]
#91 will already be included as it's merged. I just didn't list it in the changelog since it only affects test dependencies.

**Comment by jburel** [comment]
I know it is merged but I think it is helpful to have in the changelog. Changelog is not only for end user feature

**Comment by jburel** [comment]
you could even split it into
```
Features
Bugs
Others
```

**Comment by will-moore** [comment]
Done: https://github.com/will-moore/omero-mapr/blob/changelog_0.5.3/CHANGELOG.md

**Comment by will-moore** [comment]
@jburel Good to merge & release now?

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,15 @@
+0.5.3 (November 2024)
+---------------------
+
+Bug Fixes:
+
+ - Fix default favicon ([#86](https://github.com/ome/omero-mapr/pull/86))
+
+Others:
+
+ - Improved documentation (Thanks to Jens Wendt) ([#82](https://github.com/ome/omero-mapr/pull/82))
```
Comment (jburel) [inline CHANGELOG.md:10]:
```suggestion
 - Improve documentation (Thanks to Jens Wendt) ([#82](https://github.com/ome/omero-mapr/pull/82))
```

**Comment by will-moore** [comment]
@jburel Thanks for the typo fix. Can I assume that it's otherwise good to merge now?

**Comment by dominikl** [comment]
👍 Looks good to me.

**Pull request merged**